objects/client.c

```diff
@@ -723,12 +723,58 @@
  * identifier remain the same. This allow to match a spawn event to an actual
  * client.
  *
+ * This is used to display a different mouse cursor when the application is
+ * loading and also to attach some properties to the newly created client (like
+ * a `tag` or `floating` state).
+ *
+ * Some applications, like `xterm`, don't support startup notification. While
+ * not perfect, the adding the following code to `rc.lua` will mitigate the
```

Comment:
`the adding`?
Just `adding` (my preferred version) or `the addition of`

Reply:
done